format_cef/cef.py

```diff
@@ -42,16 +42,27 @@ def escape(s):
     return escape
 
 
+def ensure_in_range(debug_name, min, max, num):
+    if max is None:
+        if min is not None and num < min:
+            raise ValueError('{}: {} less than {}'.format(
+                debug_name, num, min))
+    elif min is None:
+        if max is not None and num > max:
+            raise ValueError('{}: {} greater than {}'.format(
+                debug_name, num, max))
+    elif not min <= num <= max:
+        raise ValueError('{}: {} out of range {}-{}'.format(
+            debug_name, num, min, max))
+
+
 def int_sanitiser(max=0, min=0):
```

Comment:
In both cases? I think this is really just vestigial from when it needed a range of integers to check...